system-cells/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "system-cells"
```

Comment:
Name it `ckb-system-cells`

Reply:
I think we don't need to keep the two crates. Just use the top crate for the lib and use the name `ckb-system-scripts`, and use cargo tests directory to test the scripts.